docs/10-action-optimize-media.md

```diff
@@ -0,0 +1,56 @@
+# Add an action to optimize media
+
+To add a new action in your sonata admin.
+
+![optimize-action.png](img/optimize-action.png)
+
+## Requirements
+
+- https://github.com/sonata-project/SonataMediaBundle
```

Comment:
you can remove this requirements section as it is already described in composer.json